llvm/include/llvm/IR/RepoHashCalculator.h

```diff
@@ -175,20 +175,22 @@ class HashCalculator {
   /// Return the computed hash as a string.
   std::string &get(MD5::MD5Result &HashRes);
 
-  ticketmd::DependenciesType &getDependencies() { return Dependencies; }
+  ticketmd::SmallGOVec &getDependencies() { return Dependencies; }
 
-  ticketmd::ContributionsType &getContributions() { return Contributions; }
+  ticketmd::SmallGOVec &getContributedToGVs() {
+    return ContributedToGVs;
+  }
 
 private:
   // Accumulate the hash of basicblocks, instructions and variables etc in the
   // function Fn.
   MD5 Hash;
 
-  // Hold the global object list which the function hash depenendent on.
-  ticketmd::DependenciesType Dependencies;
+  // A the global object vector which the the function/variable references.
+  ticketmd::SmallGOVec Dependencies;
 
-  // Hold the global variables which the function/variable hash contributed to.
-  ticketmd::ContributionsType Contributions;
+  // A global object vector which the function/variable hash contributed to.
```

Comment:
Maybe:
"Vector of global objects which the function/variable has contributed to"

Reply:
Fixed, thanks.